### PR TITLE
ROADS TO POWER UPDATE -- DRAFT

### PR DESCRIPTION
**Pull request by edmund-h**
## Completion Tracker:
Modified files - [use this page to find which files need updating](https://steamdb.info/patchnotes/15810853/)

VS Code native procedure for checking modified files used by  : 
1. add your name next to the folder you are working on
2. go to the page listed above and locate a file in the folder you are working on in the modified files (white text) section of the `CK3 Game Content` box
3. if you do not already have it, add the game files into your VS Code project
4. locate and right click the file in game files, select `Select for Compare`
5. if there is a corresponding file in the GoA2 files, right click it and select `Compare with Selected` (otherwise move on to the next file
6. reconcile the game file on the left (deleted text in red) to the GoA2 file on the right (added text in green)
  - if added text does not have a `#Warcraft` comment, you need to check Git Blame to see if we added it. If we did not, make an assessment as to whether the deleted content serves the mod. If you do not leave in, leave a comment stating it was removed in 1.13.x in case it causes a problem later. Otherwise, if we did add it, add a `#Warcraft` comment
  - "removed" text is not necessarily deleted by us. new material in the file will show up as "removed". generally, you should just use the arrow button to immediately add it, especially if it is related to a new feature. However, it is also a good idea to check the git blame if you are in doubt to see if and when we removed the text. If you do not add the "removed" content in, add a `# warcraft` comment with a brief reason as to why. If someone else removed the content and did not leave that comment, then you should add it.
  - feel free to ask in development chat if you are in doubt
 

**Common**

- [x] accolade_icons  @helloruhe 
- [x] accolade_names  @Grekopithikos 
- [x] achievements    @helloruhe 
- [x] activities   @Grekopithikos
- [x] ai_goaltypes   @helloruhe 
- [x] ai_war_stances   @helloruhe 
- [x] artifacts  @Grekopithikos
- [x] bookmark_portraits  
- [x] bookmarks   @helloruhe 
- [x] buildings   @Grekopithikos
- [x] casus_belli_groups   @helloruhe 
- [x] casus_belli_types  @Grekopithikos
- [x] character_backgrounds  @helloruhe 
- [x] character_interaction_categories  @Grekopithikos
- [x] character_interactions  @Grekopithikos
- [x] character_memory_types  @helloruhe 
- [x] coat_of_arms  @helloruhe 
- [x] combat_effects  @helloruhe 
- [x] combat_phase_events  @helloruhe 
- [x] console_groups  @helloruhe 
- [x] council_positions  @helloruhe 
- [x] council_tasks  @helloruhe 
- [x] court_positions  @helloruhe 
- [x] court_types  @helloruhe 
- [x] courtier_guest_management  @helloruhe 
- [x] culture  @Grekopithikos
- [x] customizable_localization  @helloruhe 
- [x] deathreasons  @helloruhe 
- [x] decisions   @helloruhe 
- [x] defines   @helloruhe 
- [x] dna_data   @helloruhe 
- [x] dynasties   @helloruhe 
- [x] dynasty_house_motto_inserts   @helloruhe 
- [x] dynasty_house_mottos   @helloruhe 
- [x] dynasty_houses   @helloruhe 
- [x] dynasty_legacies   @helloruhe 
- [x] dynasty_perks   @helloruhe 
- [x] effect_localization   @helloruhe 
- [x] ethnicities  @helloruhe 
- [x] event_backgrounds   @helloruhe 
- [x] event_themes   @helloruhe 
- [x] factions  @helloruhe 
- [x] flavorization  @helloruhe 
- [x] game_concepts  @helloruhe 
- [x] game_rules  @helloruhe 
- [x] genes  @helloruhe
- [x] gfx @helloruhe
- [x] governments @helloruhe  
- [x] holdings   @helloruhe
- [x] hook_types   @helloruhe
- [x] house_unities   @helloruhe
- [x] important_actions   @helloruhe
- [x] inspirations   @helloruhe
- [x] landed_titles   @helloruhe
- [x] laws   @helloruhe
- [x] lifestyle_perks  @helloruhe 
- [x] men_at_arms_types  @helloruhe 
- [x] messages  @helloruhe 
- [x] modifier_definition_formats   @helloruhe 
- [x] modifiers   @helloruhe 
- [x] named_colors @helloruhe  
- [x] nicknames  @helloruhe 
- [x] on_action  @helloruhe 
- [x] opinion_modifiers  @helloruhe 
- [x] pool_character_selectors  @helloruhe 
- [x] province_terrain  @helloruhe 
- [x] religion  @Grekopithikos 
- [x] schemes  @helloruhe 
- [x] script_values  @helloruhe 
- [x] scripted_animations  @Grekopithikos 
- [x] scripted_character_templates  @edmund-h 
- [x] scripted_effects  @edmund-h 
- [ ] scripted_guis  @edmund-h 
- [x] scripted_modifiers  @Grekopithikos 
- [x] scripted_relations  @edmund-h 
- [x] scripted_rules  @edmund-h 
- [x] scripted_triggers  @edmund-h 
- [x] secret_types    @edmund-h 
- [x] story_cycles    @edmund-h 
- [x] struggle    @edmund-h 
- [x] succession_election  @edmund-h 
- [x] suggestions @edmund-h  
- [x] traits  @Grekopithikos
- [x] travel  @edmund-h 
- [x] trigger_localization  @edmund-h 
- [x] tutorial_lesson_chains  @edmund-h 
- [x] tutorial_lessons  @edmund-h 
- [x] vassal_contracts  @edmund-h  
- [x] vassal_stances  @edmund-h 

**Events**

- [x]  activities @helloruhe
- [x]  artifacts @helloruhe
- [x]  councilor_task_events @helloruhe
- [x]  court_events @helloruhe
- [x]  courtier_guest_management_events
- [x]  culture_events @Grekopithikos
- [x]  death_events @Grekopithikos
- [x]  decisions_events @Grekopithikos
- [x]  diarchy_events @Grekopithikos
- [x]  dlc @Grekopithikos
- [x]  education_and_childhood @Grekopithikos
- [x]  factions @Grekopithikos
- [x]  government_events @helloruhe
- [x]  interaction_events @helloruhe
- [x]  lifestyles @Grekopithikos
- [x]  nickname_events @Grekopithikos
- [x]  prison_events @Grekopithikos
- [x]  relations_events @helloruhe
- [x]  religion_events @helloruhe
- [x]  scheme_events @helloruhe
- [x]  story_cycles @Grekopithikos
- [x]  stress_events @Grekopithikos
- [x]  test_events @Grekopithikos
- [x]  trait_specific_events @Grekopithikos
- [x]  travel_events @Grekopithikos
- [x]  war_events @helloruhe
- [x]  yearly_events @Grekopithikos
- [x]  [new top level files] @helloruhe
- [x]  wc_events fixes @helloruhe

**GFX**

- [x]  coat_of_arms @Chomski2
- [x]  court_scene @Chomski2
- [x]  FX @Chomski2
- [x]  interface @Chomski2
- [x]  map @Chomski2
- [x] models @Chomski2 / @helloruhe
- [x]  portraits @Chomski2 / @helloruhe
- [x]  skins @Chomski2

**GUI** @helloruhe 
- [x] custom_gui  
- [x] decision_view_widgets  
- [x] preload  
- [x] scripted_widgets  
- [x] shared  
- [x] root dir  

**Localization**
- [ ] replace > english > culture
- [ ] replace > english > dlc
- [x] replace > english > event_localization @helloruhe
- [ ] replace > english > gui
- [ ] replace > english > names
- [ ] [new needed localization]

**Other Tasks**
- [ ] Add new notification types to all event toasts in WC events
- [ ] Test government change resulting from a necro/demon/eldritch faction [ DM @edmund-h  for details ]
- [x] Redo clothing style triggers @helloruhe 
- [ ] replace all death with override death effect in non-WC events (ongoing task)
- [ ] test plague scheme - major changes were made to schemes and this needed to be completely redone
- [x] replace new loading screen @helloruhe 
- [ ] Must Adapt as in #1482, and some of the Can Adapt if time permits
- [ ] Fix everything in #1492 
- [ ] Change age to Age (Relative to Humans)
- [ ] clean error log

### CODE REVIEW

File: common/casus_belli_types/wc_event_wars.txt
Comment (helloruhe) [inline common/casus_belli_types/wc_event_wars.txt:1]: why was this removed

Comment (edmund-h) [reply common/casus_belli_types/wc_event_wars.txt:1]: it wasnt, it was never added

Comment (edmund-h) [reply common/casus_belli_types/wc_event_wars.txt:1]: i added it to resolve an error rather than deleting everything related. it doesnt have anything to do with marco polo it creates a book called The Travels of [FirstName] [LastName]

**Comment by edmund-h** [comment]
@helloruhe in the future please remove draft title before merging ty
